cmd/hhfab/main.go

```diff
@@ -881,6 +881,12 @@ func Run(ctx context.Context) error {
 								Aliases: []string{"mtu"},
 								Usage:   "interface MTU for VPCs advertised by DHCP",
 							},
+							&cli.StringFlag{
+								Name:    "hash-policy",
+								Aliases: []string{"hash"},
+								Usage:   "xmit_hash_policy for bond interfaces on servers [layer2|layer2+3|layer3+4|encap2+3|encap3+4|vlan+srcmac]",
```

Comment:
```suggestion
								Usage:   "xmit_hash_policy for bond interfaces on servers [layer2|layer2+3]",
``` 
we only support the hashing policy that are 802.3ad compliant. Truth be told, I am not sure how / when the other policies are deployable, presumably when the switch and NIC are from the same manufacturer or testing has been done.

Reply:
as discussed offline, I've kept the other modes but added a warning if a non fully compliant option is selected. I've also exposed the hash-policy flag in the release-test so we can run tests with each of these and determine which ones are working (as part of https://github.com/githedgehog/fabricator/issues/559)